packages/components/src/buttons/src/Button.module.css

```diff
@@ -67,15 +67,15 @@
     /* Danger */
     --hop-Button-danger-color: var(--hop-danger-text-strong);
     --hop-Button-danger-color-hover: var(--hop-danger-text-strong-hover);
-    --hop-Button-danger-color-pressed: var(--hop-danger-text-active);
+    --hop-Button-danger-color-pressed: var(--hop-danger-text-strong-hover);
     --hop-Button-danger-color-disabled: var(--hop-neutral-text-disabled);
     --hop-Button-danger-background-color: var(--hop-danger-surface-strong);
-    --hop-Button-danger-background-color-hover: var(--hop-danger-surface-hover);
-    --hop-Button-danger-background-color-pressed: var(--hop-danger-surface-active);
+    --hop-Button-danger-background-color-hover: var(--hop-danger-surface-strong-hover);
```

Comment:
just making sure this is intentional. Was it for the contrast issues?

Reply:
This was to fix an issue where the pressed color was the same as the default 
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/9817fe91-25d2-42ed-a1fb-5a5196917052)